tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/BaseOnboard.kt

```diff
@@ -0,0 +1,284 @@
+package net.corda.cli.plugins.mgm
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import kong.unirest.HttpResponse
+import kong.unirest.Unirest
+import kong.unirest.json.JSONObject
+import net.corda.crypto.test.certificates.generation.CertificateAuthorityFactory
+import net.corda.crypto.test.certificates.generation.toFactoryDefinitions
+import net.corda.crypto.test.certificates.generation.toPem
+import net.corda.v5.cipher.suite.schemes.RSA_TEMPLATE
+import org.bouncycastle.openssl.PEMParser
+import org.bouncycastle.pkcs.PKCS10CertificationRequest
+import picocli.CommandLine.Option
+import picocli.CommandLine.Parameters
+import java.io.File
+import java.io.InputStream
+import java.net.ServerSocket
+import kotlin.concurrent.thread
+
+abstract class BaseOnboard : Runnable {
+    private companion object {
+        const val P2P_ALIAS = "p2p-tls-cert"
+        const val P2P_CERTIFICATE_ALIAS = "p2p-tls-cert"
+    }
+
+    @Parameters(
+        description = ["The name of the k8s namespace (leave empty for combined worker on local host)"],
+        paramLabel = "NAME",
+        arity = "0..1",
+    )
+    var networkName: String? = null
```

Comment:
```suggestion
    var namespace: String? = null
```
Maybe `namespace` matches the purpose a bit better?

Reply:
```suggestion
    var cordaCluster: String? = null
```
Or `cordaCluster` to be more consistent with other changes?